integration-tests/cli/product-json-to-csv.it.js

```diff
@@ -1,6 +1,6 @@
 import fs from 'mz/fs'
 import tmp from 'tmp'
-import unzip from 'unzip'
+import unzipper from 'unzipper'
```

Comment:
`unzip` is not maintained and does not even work in node 12 anymore. `unzipper` is a maintained fork of it and works fine.